lib/custom_widgets/movie_gesture_detector.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+
+import '../entities/movie.dart';
+import '../text_styles/custom_text_style.dart';
+import '../ui_constants.dart';
+import '../views/movie_details_view.dart';
+
+class MovieGestureDetector extends StatelessWidget {
```

Comment:
i think we could find another name for this class , something more related to what the widget contains , not its functionality. Something like MovieInfo maybe?